### PR TITLE
[Raleigh-2025] Added an Organizer and two Community Sponsors

### PR DESCRIPTION
**Pull request by jrn00**
Added Shawn Taylor as an Organizer
     - Added picture
    - Added info in RDU main.yml
    
Added KubeEvents and KubeCareers under Community in main.yml

### CODE REVIEW

File: static/events/2025-raleigh/organizers/shawn-taylor.jpg
Comment (toshywoshy) [inline static/events/2025-raleigh/organizers/shawn-taylor.jpg:1]: Please add picture to the assets folder
`assets/events/2025-raleigh/organizers/`

**Review by toshywoshy — CHANGES_REQUESTED** [review]

**Comment by toshywoshy** [comment]
Adding a new organizer requires @core to approve mailinglist changes and the new organizer

File: static/events/2025-raleigh/organizers/shawn-taylor.jpg
Comment (jrn00) [reply static/events/2025-raleigh/organizers/shawn-taylor.jpg:1]: I've added his picture to static/events/2025-raleigh/organizers

That's where the rest of our portraits reside. 

Was there a change?

Comment (jrn00) [reply static/events/2025-raleigh/organizers/shawn-taylor.jpg:1]: Also, email sent to Core

Comment (nimbinatus) [reply static/events/2025-raleigh/organizers/shawn-taylor.jpg:1]: Hey Jeremy! I'm going through and doing the organizer update now. Yes, images are being moved to the assets directory (https://github.com/devopsdays/devopsdays-web/pull/14341). At the moment, both *should* work, iirc, and should not be a breaking change, but we're working on getting folks to get used to the new workflow.

That being said, there's something up with the preview. No events are showing, and I can't get to the logs to find out what's up. I'll pull it down here in a moment and figure out what's going on.

**Review by nimbinatus — DISMISSED** [review]
Found the break, and a couple other things. I'll reply to the email with the distro update confirmation in a minute.

**Comment by nimbinatus** [comment]
Core updates complete; once you've gotten this PR patched, anyone can review and merge (fyi to @devopsdays/content-reviewers )

File: static/events/2025-raleigh/organizers/shawn-taylor.jpg
Comment (jrn00) [reply static/events/2025-raleigh/organizers/shawn-taylor.jpg:1]: Thanks Laura!

I made those suggested Changes. 

I can move our pics over to the assets directory here soon.

**Review by toshywoshy — APPROVED** [review]

**Review by toshywoshy — APPROVED** [review]

**Pull request merged**